app/javascript/src/components/Dashboard/Contacts/constants.js

```diff
@@ -1,9 +1,10 @@
 import React from "react";
 
+import dayjs from "dayjs";
 import * as yup from "yup";
 
-import ContactAvatar from "./Contact/ContactAvatar";
-import ContactDetailText from "./Contact/ContactDetailText";
+import Avatar from "./Contact/Avatar";
+import DetailText from "./Contact/DetailText";
```

Comment:
Given that there are multiple modules being imported from same namespace, add an `index.js` file within `Contact` and export all necessary modules.

Reply:
I have made the changes and have merged the PR.